pyrt/renderer/simplert.py

```diff
@@ -50,17 +50,21 @@ def _shade(self, scene: Scene, ray: Ray, hitrecord: HitRecord) -> tuple:
 
     def _shadow(self, scene: Scene, hitrecord: HitRecord) -> tuple:
         # is hitpoint in shadow ?
-        fs = 1.0
+        fs = 0.0
         local_num_shadow_rays = 0
         numshadow = 0
         for light in scene.lights:
             shadowray = Ray(hitrecord.point, light.position - hitrecord.point)
+            fs += light.intensity(shadowray) 
+
             local_num_shadow_rays += 1
             for testelement in scene.nodes:
                 if testelement != hitrecord.obj:  # avoid self-intersection
                     if testelement.hitShadow(shadowray):
                         numshadow += 1
                         break
+
+        fs = np.clip(fs, 0, 1)
```

Comment:
Was it been tested with point light again?

Reply:
Yes